internal/plugins/helm/v1/scaffolds/templates/manager/config.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I need to review all files that are with our license but are a copy from KB and are modified to add the kb license + modifications one such as:  uphttps://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications